src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033026 "lung perichondrial fibroblast")
 EquivalentClasses(obo:CL_4033026 ObjectIntersectionOf(obo:CL_4033025 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002048)))
 
+# Class: obo:CL_4033027 (diffuse bipolar 1 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22006647") obo:IAO_0000115 obo:CL_4033027 "An OFF diffuse bipolar cell that makes synaptic contact with both L/M and S-cone photoreceptors, and only minimal contact with rod photoreceptors.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033027 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033027 "2023-04-04T10:48:29Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033027 "DB1 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22006647") Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033027 "A diffuse bipolar 1 cell on average makes 81 ribbon (output) synapses, 13 contacts are with OFF parasol cells and 47 contacts are with OFF midget cells. This cell might make flat contacts at both triad-associated and nontriad-associated positions on the cone pedicle.")
```

Comment:
The numbers quoted are related to [a specific review](https://www.sciencedirect.com/science/article/pii/S1350946220300161?via%3Dihub#bib391). Are these numbers generalisable and applicable across all (or at least many) species? If not, I recommend removing the comment or identifying to which species they apply. 

If keeping, consider replacing "nontriad" with "non-triad".

Reply:
Thank you for the suggestion.